### DISCUSSION TITLE
Error in making and testing moose

### DISCUSSION THREAD

**Discussion by AmbroiseJuston** [idaholab/moose#26134] [Q&A Getting Started] [answered]
I have been trying to compile and test moose onto my windows machine using WSL. I have followed the steps provided at https://mooseframework.inl.gov/getting_started/installation/conda.html under getting started. I went all through cloning MOOse and every step seemed to work however when it comes to building and testing moose I keep getting the same error (see below).

Error in question:
![image](https://github.com/idaholab/moose/assets/116405013/987eadf0-0943-4749-8e5e-bac2343c7bd2)

```
Compiling C++ (in opt mode) /home/ambroisej/projects/moose/test/build/unity_src/dgkernels_Unity.C...
{standard input}: Assembler messages:
{standard input}: Error: open CFI at the end of file; missing .cfi_endproc directive
{standard input}:457952: Error: invalid operands (*UND* and .gcc_except_table._ZN31PostprocessorInterfaceErrorTest11validParamsEv sections) for `-'
x86_64-conda-linux-gnu-c++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
make: *** [/home/ambroisej/projects/moose/framework/build.mk:150: /home/ambroisej/projects/moose/test/build/unity_src/userobjects_Unity.x86_64-conda-linux-gnu.opt.lo] Error 1
make: *** Waiting for unfinished jobs....
```

I copied and pasted the commands provided under that section however when running make -j 6 it will compile for a couple of minutes before returning the error shown above. The first time I ran this my laptop died so I initially just reran it while restarting from the top however the error persisted. I also assumed it could be from lack of space on my disk however I have around 120Gb of free memory left so I am not sure.

any help would be appreciated, thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Can you try `make -j 1` ?
Kill signals often happen when you run out of memory. and -j6 will use quite a bit

Guillaume

**Reply by AmbroiseJuston** [reply]
It was indeed simply this. I thought I had tried, but I guess not. In any case, thank you very much for the help; it is appreciated.

**Comment by phillipbuelow** [comment]
Appreciate the insight. I ran into the same error and resolved it through `make -j 3`
